packages/twenty-server/src/metadata/data-source/data-source.entity.ts

```diff
@@ -26,12 +29,12 @@ export class DataSourceEntity {
   @Column({ type: 'enum', enum: ['postgres'], default: 'postgres' })
   type: DataSourceType;
 
-  @Column({ nullable: true, name: 'label' })
-  label: string;
-
   @Column({ default: false })
   isRemote: boolean;
 
+  // @Column({ default: false })
```

Comment:
Actually it's still a WIP, I was thinking on a way to distinguish system datasource like `core` and other datasource, because I don't think we want to expose system datasources